### PR TITLE
Copy edits

### PR DESCRIPTION
**Pull request by kberzinch**
Summary of changes:
- Add preamble that describes the document
- Switch to straight quotes
- Various edits for clarity

### CODE REVIEW

**Review by JoshOldenburg — COMMENTED** [review]
Is it possible to compress it slightly so that the parent/guardian signature is not on a separate printed page? That may require a change in RoboJackets/apiary instead of this document.

The text looks good to me.

**Comment by kberzinch** [comment]
Yes that's in the blade file for printing. Feel free to add it to https://github.com/RoboJackets/apiary/issues/1561

**Review by JoshOldenburg — DISMISSED** [review]
Will do.

File: agreement.md [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+Guardian and Participant, and Guardian permits Participant to participate and engage in all activities related to
+RoboJackets.
 @endif
 
 1. **Waiver and Release:** I, the Participant, hereby release, to the fullest extent permitted by law, RoboJackets, Inc.
-and all of its directors, officers, and agents (herein referred to collectively as “RoboJackets”) from any and all
-liability, claims, demands, injuries, damages and causes of action arising from or relating to my participation in
+and all of its directors, officers, and agents (herein referred to collectively as "RoboJackets") from any and all
+liability, claims, demands, injuries, damages, and causes of action arising from or relating to my participation in
 activities and travel in connection with RoboJackets.
```
Comment (ryanstrat) [inline agreement.md:26]: This is missing the new "Damage Waiver" clause I wrote yesterday.

> Damage Waiver: In no event shall RoboJackets be liable under this agreement for any special, consequential, or incidental damages of any kind arising from my participation in activities and travel in connection with RoboJackets.

Comment (kberzinch) [reply agreement.md:26]: This is included in lines 27-28.

**Pull request merged**